### PR TITLE
Rounding

### PR DESCRIPTION
**Pull request by jnelson16**

### CODE REVIEW

File: quantgov/__main__.py [hunk 1]
```diff
@@ -118,6 +118,9 @@ def parse_args():
     estimate.add_argument(
         '--probability', action='store_true',
         help='output probabilities instead of predictions')
+    estimate.add_argument(
+        '--precision', default=4,
```
Comment (OliverSherouse) [inline quantgov/__main__.py]: Set type to int here rather than cast in the function itself.

File: quantgov/estimator/estimation.py [hunk 2]
```diff
@@ -117,10 +118,12 @@ def estimate_probability_multiclass(vectorizer, model, streamer):
     """
     pipeline = get_pipeline(vectorizer, model)
     texts = (doc.text for doc in streamer)
-    yield from zip(streamer.index, pipeline.predict_proba(texts))
+    yield from zip(streamer.index, (i.round(int(precision))
+                   for i in pipeline.predict_proba(texts)))
```
Comment (OliverSherouse) [inline quantgov/estimator/estimation.py]: Round at the array level, not the obvservation level. So:

```yield from zip(streamer.index,  pipeline.predict_proba(texts).round(precision))```

File: quantgov/estimator/estimation.py [hunk 3]
```diff
@@ -137,7 +140,7 @@ def estimate_probability_multilabel_multiclass(vectorizer, model, streamer):
     texts = (doc.text for doc in streamer)
     predicted = pipeline.predict_proba(texts)
     for i, docidx in enumerate(streamer.index):
-        yield docidx, tuple(label_predictions[i]
+        yield docidx, tuple(label_predictions[i].round(int(precision))
```
Comment (OliverSherouse) [inline quantgov/estimator/estimation.py]: See above.

**Comment by jnelson16** [comment]
@OliverSherouse fixed those changes

File: quantgov/estimator/estimation.py [hunk 4]
```diff
@@ -61,7 +61,7 @@ def estimate_probability(vectorizer, model, streamer, precision):
     pipeline = get_pipeline(vectorizer, model)
     texts = (doc.text for doc in streamer)
     truecol = list(int(i) for i in model.model.classes_).index(1)
-    predicted = ((round(i[truecol], int(precision))
+    predicted = ((round(i[truecol], precision)
```
Comment (OliverSherouse) [inline quantgov/estimator/estimation.py]: Why aren't we selecting the whole column and rounding at once? I think that would be `pipeline.predict_proba[:,truecol].round(precision)`.

**Pull request merged**